lib/note-content-editor.tsx

```diff
@@ -644,7 +663,7 @@ class NoteContentEditor extends Component<Props> {
     editor.onDidChangeModelContent(() => this.setDecorators());
 
     document.oncopy = (event) => {
-      // @TODO: This is selecting everything in the app but we should only
+      // @TODO: This is selecting everything in the app but wdde should only
```

Comment:
nice fix!

Reply:
haha, was testing spellcheckers in VS Code. super meta. :)